pages/aptos/sdks.mdx

```diff
@@ -0,0 +1,40 @@
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# SDKs
+
+Aptos contracts can program against the Pyth contract's interface by including it as a project dependency.
+
+## Installation Instructions
+
+<Tabs items={['Move']}>
+    <Tab>
+        First, add the Pyth contract to your project dependencies by adding the following line to the `[dependencies]` section of `Move.toml`:
```

Comment:
I suggest mentioning the cli version that this contract was compiled with:
https://github.com/aptos-labs/aptos-core/releases/download/aptos-cli-v1.0.4/aptos-cli-1.0.4-Ubuntu-22.04-x86_64.zip

The latest version of the cli is far from backward compatible and may not work as expected